packages/default-storage/windows/code/codegen/.clang-format

```diff
@@ -0,0 +1,2 @@
+DisableFormat: true
+SortIncludes: false
```

Comment:
Add newline to the end of file: https://www.baeldung.com/linux/files-end-with-newlines